boards/mediatek/mt8186/mt8186_adsp.dts

```diff
@@ -37,12 +37,12 @@
 			#interrupt-cells = <3>;
 		};
 
-		intc2: intc@10680010 {
```

Comment:
some spelling mistakes: becuase->because, "process on thsi device, the and the set"